cf-nodejs-logging-support-core/log-core.js

```diff
@@ -438,6 +438,27 @@ var getCorrelationId = function () {
     return null;
 };
 
+//setTenantId sets the tenant_id for the logger this is called on. Returns true if set, false otherwise
+var setTenantId = function (tenantId) {
+    var logger = this;
+    if (logger.logObject != null) {
+        logger.logObject.tenant_id = tenantId;
+        return true;
+    }
+    return false;
+};
```

Comment:
As you can see in this [documentation of the java logging library](https://github.com/SAP/cf-java-logging-support/blob/master/cf-java-logging-support-core/beats/request-metrics/docs/fields.asciidoc)
(which the fields present here are based on)
the tenant_id needs to be a uuid.
therefore I would like to have this change change:
```suggestion
var setTenantId = function (tenantId) {
    var logger = this;
    if (logger.logObject != null) {
        if (uuidCheck.exec(tenantId)) {
            logger.logObject.tenant_id = tenantId;
            return true;
        }
    }
    return false;
};
```